semgrep/semgrep/config_resolver.py

```diff
@@ -27,6 +27,7 @@
 IN_DOCKER = "SEMGREP_IN_DOCKER" in os.environ
 IN_GH_ACTION = "GITHUB_WORKSPACE" in os.environ
 REPO_HOME_DOCKER = "/home/repo/"
+REPO_HOME_DOCKER_PRECOMMIT = "/src/"
```

Comment:
Here was the big blocker that was breaking `pre-commit` - it mounts the code at `/src/`. We can re-purpose the `--precommit` flag to handle this case.